dockerfiles/arch-linux.dockerfile

```diff
@@ -17,8 +17,7 @@
 FROM archlinux
 
 RUN \
-  pacman --sync --noconfirm --refresh --sysupgrade && \
-  pacman --sync --noconfirm \
+  pacman -Syyu --noconfirm \
```

Comment:
Can we use long options for readability?

Reply:
I see.

Reply:
I've fixed it.